### PR TITLE
I1258 b update release notes

### LINKED ISSUE #1258: Update Open GEE 5.3.0 Release Notes
Update Open GEE 5.3.0 Release Notes

### PR DESCRIPTION
**Pull request by tst-eclamar**
The is a second attempt to push changes to 7160007.html - update release notes.

Fixes #1258 

Release notes can be viewed at:
https://tst-eclamar.github.io/earthenterprise/geedocs/5.3.0/answer/7160007.html

### CODE REVIEW

**Comment by tst-nfarah** [comment]
Let's push this to github docs for easier review

**Comment by tst-lsavoie** [comment]
It looks like our known issues have gone stale. It may be time to update those.

**Comment by tst-lsavoie** [comment]
The description for resolved issue #171 is confusing. I think it would be best to change it to the issue title (Add advanced parameters to geserver installer).

**Comment by tst-lsavoie** [comment]
Nitpick: on resolved issue #126 we can delete the phrase "No current workaround." since the workaround is listed right after that sentence.

**Comment by tst-lsavoie** [comment]
The workaround for known issue #343 is missing a letter: `Void opening files with .kip extension.` --> `Avoid opening files with .kip extension.`

**Comment by tst-lsavoie** [comment]
Resolved issue #279 is out of order and should be moved up so that the resolved issues are in alphabetical order.

**Comment by tst-lsavoie** [comment]
I think issue #171 merits inclusion as a new feature (i.e., it should be moved to the top rather than the resolved issues section).

**Comment by tst-lsavoie** [comment]
After looking through the notes here's what seems to be missing:

New features
Issue #1020
Issue #1192
Issue #1275

Resolved issues
Issue #1104
Issue #1169
Issue #1201
Issue #1276

**Comment by tst-eclamar** [comment]
@tst-lsavoie  what items are out of order?  I can't find any...  Are you looking at https://tst-eclamar.github.io/earthenterprise/geedocs/5.3.0/answer/7160007.html or an older webpage?

**Comment by tst-lsavoie** [comment]
Hm, I must have been looking at the wrong one. Let me take another look.

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 1]
```diff
@@ -49,6 +49,16 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+    <p><strong>Added PUBLISHER_ROOT size checking.</strong>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/7160007.html]: I would reword this something like the following:
```
    <p><strong>Added options to the Server install script</strong>
       The server install script now checks the size of the publish root and allows the user to set custom user
       and group names.
</p>
```

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 2]
```diff
@@ -49,6 +49,16 @@ <h5>
...
+       ASSET_ROOT checking is performed in fusion installer - not duplicated in server installer.
+       Added <code>-pgu pguser</code>, <code>-au apacheuser</code>, and <code>-g group</code> options.
+       Added prompt and summary output similar to fusion installer.</p>
+
+    <p><strong>We have improved the performance of large builds</strong> by consolidating some of the notifications that an asset's child or input has changed. By preventing (some) duplicate notifications we reduce the number of times each asset has to calculate its state, significantly improving performance.
+
+    <p><strong>Google has developed updated kmlrender libraries.</strong></p>
```
Comment (tst-nfarah) [inline docs/geedocs/5.3.0/answer/7160007.html]: Is this phrase incomplete ? It mentioned that Google has... but it does not state what is new in 5.3.0 as far as those kml libraries. Did you meant we have integrated a new version of kmlrender libraries ?

Comment (tst-eclamar) [reply docs/geedocs/5.3.0/answer/7160007.html]: I don't know - the PR that this refers to had very little information in it.  See https://github.com/google/earthenterprise/issues/1275.  I'll take any recommendations.

Comment (tst-nfarah) [reply docs/geedocs/5.3.0/answer/7160007.html]: The PR seems to refers to the use of a new/updated kml libraries used to render KML files in Portable. I assume the reason is performance and may be bug fixes @tst-dsheremata ?

Comment (tst-lsavoie) [reply docs/geedocs/5.3.0/answer/7160007.html]: I was involved in that work so I can write up something when I get a few minutes.

Comment (tst-lsavoie) [reply docs/geedocs/5.3.0/answer/7160007.html]: This might go better in the resolved issues section. The text would be something like this:

Description: Search results from KML layers do not display properly in the Earth Client.

Resolution: Updated KML rendering libraries with new versions provided by Google.

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 3]
```diff
@@ -49,6 +49,16 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+    <p><strong>Added PUBLISHER_ROOT size checking.</strong>
+       ASSET_ROOT checking is performed in fusion installer - not duplicated in server installer.
+       Added <code>-pgu pguser</code>, <code>-au apacheuser</code>, and <code>-g group</code> options.
+       Added prompt and summary output similar to fusion installer.</p>
+
+    <p><strong>We have improved the performance of large builds</strong> by consolidating some of the notifications that an asset's child or input has changed. By preventing (some) duplicate notifications we reduce the number of times each asset has to calculate its state, significantly improving performance.
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/7160007.html]: I think this is too low level for release notes. It would read better something like this:

<p><strong>Improved system manager performance for large builds.</strong>
This release eliminates some duplicate processing in system manager, which reduces the amount of time it takes to initiate large builds.

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 4]
```diff
@@ -103,32 +97,103 @@ <h5>
...
+          </tr>
+          <tr>
+            <td>1122</td>
+            <td>Check for PIL (Python Imager Library) returns an error</td>
+            <td>
+              Change how PIL is included.
+            </td>
+          </tr>
+          <tr>
+            <td>1138</td>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/7160007.html]: I would leave this one out since it was introduced and fixed in the latest release.

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 5]
```diff
@@ -103,32 +97,103 @@ <h5>
...
             <td>
-              Help has been added through the <code>--help</code> option, or if any invalid parameter is given
+              Use POSIX file API to support large files on some 32-bit platforms.
+            </td>
+          </tr>
+          <tr>
+            <td>1192</td>
+            <td>There appears to be a regression issue in OpenJpeg 2000 lib version 2.3.0 (that Fusion uses), causing an error when fusing jp2 files.</td>
+            <td>
+               This OpenJpeg 2000 issue is fixed in OpenJpeg master but since the fix was made there was no new released version of OpenJpeg 2000 library (as per today: 2/8/2019).
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/7160007.html]: This is no longer accurate as OpenJpeg 2.3.1 has been released. We can change this to just say that we upgraded to OpenJpeg 2.3.1.

**Comment by tst-lsavoie** [comment]
Here's two issues that I think could be added to the known issues, together with a rough cut at a workaround:

Issue #1257: Ensure you use a valid date when creating resources from the command line
Issue #1202: Add the resource from the command line

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 6]
```diff
@@ -49,6 +49,16 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+<p><strong>Added options to the Server install script</strong>
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/7160007.html]: Minor: need a period at the end of this sentence.

File: docs/geedocs/5.3.0/answer/7160007.html [hunk 7]
```diff
@@ -49,6 +49,14 @@ <h5>
...
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+<p><strong>Added options to the Server install script.</strong>
+       The server install script now checks the size of the publish root and allows the user to set custom user and group names.
+</p>
+
+<p><strong>Improved system manager performance for large builds.</strong> This release eliminates some duplicate processing in system manager, which reduces the amount of time it takes to initiate large builds.
+</p>
+
```
Comment (tst-nfarah) [inline docs/geedocs/5.3.0/answer/7160007.html]: We probably should mention the upgrade of OpenJpeg 2000 library (beside the bug fix) part of new features or a section by itself for Updated libraries. We could mention that upgrade bring `bugs fix and improvements made into that new version`
